script/release

```diff
@@ -326,7 +326,7 @@ def main():
 
     # push package to pypi
     step(f"Generate and push package to {args.repository or args.repository_url or 'pypi'}")
-    execute(["python", "setup.py", "bdist_wheel", "sdist"], log=True)
+    execute(["python", "-m", "build"], log=True)
```

Comment:
how is `build` installed? I'm missing the line where it's added to dependencies somewhere or installed as part of the release step